project_functional_block/models/project_functional_block.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2010 Akretion LDTA (<http://www.akretion.com>).
+#
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api
+
+
+class ProjectFunctionalBlock(models.Model):
+    _name = 'project.functional.block'
+    _description = 'Functional block to organize projects tasks'
+    _order = 'complete_name'
+    _rec_name = 'complete_name'
+
+    # Columns
+    name = fields.Char('Name', size=64, required=True, translate=True)
+    code = fields.Char('Code', size=8)
```

Comment:
I am not sure that it is the best practice. For me, and also in my company, the best practice is to specifying explicitly the size of each `Char` field so that
- We have a clear understanding how much characters we can input in a field
- We can reduce the size of the database.

I will wait for your feedback to do this modification.